package.json

```diff
@@ -29,7 +29,7 @@
     "mime": "1.3.6",
     "mongodb": "2.2.26",
     "multer": "1.3.0",
-    "parse": "1.9.2",
+    "parse": "eduardbosch/Parse-SDK-JS#contains-all-starting-with",
```

Comment:
Since you are using REST the JavaScript branch isn't needed. Changing it back will remove the conflict also.